csrc/quantization/awq/gemm_kernels.cu

```diff
@@ -493,9 +493,117 @@ __global__ void __launch_bounds__(64) gemm_forward_4bit_cuda_m16n64k32(int G, in
 #endif
 }
 
+__global__ void __launch_bounds__(64) dequantize_weights(
```

Comment:
Just wondering: is this also supported by T4?

Reply:
Yes it should be. The CC 7.5 support was only for the mma.sync.aligned.m16n8k16 instruction. This should be compatible as it’s already in the GEMM kernel.

Reply:
Sounds good! Thanks for the clarification.